dashboard/src/App.js

```diff
@@ -47,25 +55,36 @@ const HomeRoute = ({ redirectPath = APP_ROUTES.HOME }) => {
 
 const App = () => {
   const dispatch = useDispatch();
+  const { keycloak } = useSelector(
+    state => state.apiEndpoint
+  );
 
   useEffect(() => {
     const faviconLogo = document.getElementById("favicon");
     faviconLogo?.setAttribute("href", favicon);
 
     dispatch(fetchEndpoints);
-    dispatch(getUserDetails());
   }, [dispatch]);
 
   return (
     <div className="App">
+    { keycloak && (
```

Comment:
If you say so ... 😆 That "it will just keep loading" doesn't seem obvious from the code here. As long as we're seeing "live interface" and not just an empty component, S'OK. If there's a delay in "becoming functional" we should have a visual indicator (like the spinner).